core/kazoo_attachments/src/kz_att_http.erl

```diff
@@ -82,7 +82,9 @@ format_verb(<<"post">>) ->
 format_verb(<<"PUT">>) ->
     'put';
 format_verb(<<"put">>) ->
-    'put'.
+    'put';
+format_verb(Value) ->
```

Comment:
@sergey-safarov what `verb` are you trying to handle besides `POST`and `PUT` ?

Reply:
In my case i got crash because `format_verb` is called with `put` argument as `text, not as `binary`.
I will add one commit.

Reply:
@sergey-safarov `Verb` comes from settings and it should be a `binary` not `text`

Reply:
`Account/tenant` or `system_config` database?
I have defined recording at account object like
```json
  "call_recording": {
    "endpoint": {
      "inbound": {
        "onnet": {
          "enabled": true,
          "time_limit": 3600,
          "url": "http://217.12.247.220",
          "format": "mp3"
        }
      }
    }
  }
```
and get crash
one more commit is pushed.